src/styles/blocks/header.scss

```diff
@@ -0,0 +1,13 @@
+.header {
+  background: $header-background;
+  display: flex;
+  justify-content: space-between;
```

Comment:
To ensure that there are always 4 cards in a row, you can use 'grid-template-columns' property with 'repeat' function for the parent container.